docs/conf.py

```diff
@@ -1,11 +1,18 @@
+comments_config = {
+    "hypothesis": True,
+    "dokieli": True,
```

Comment:
I would say either dokieli or hypothesis, because otherwise there are too many buttons over each other. Since we already use hypothesis, remove dokieli.